src/commands/aksEraserTool/erasertool.ts

```diff
@@ -0,0 +1,86 @@
+import * as vscode from "vscode";
+import * as k8s from "vscode-kubernetes-tools-api";
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { getAksClusterTreeNode, getClusterProperties, getKubeconfigYaml } from "../utils/clusters";
+import { Errorable, failed, succeeded } from "../utils/errorable";
+import { longRunning } from "../utils/host";
+import { invokeKubectlCommand } from "../utils/kubectl";
+import * as tmpfile from "../utils/tempfile";
+
+
+export default async function aksEraserTool(_context: IActionContext, target: unknown): Promise<void> {
+    const cloudExplorer = await k8s.extension.cloudExplorer.v1;
```

Comment:
To support getting the kubeconfig from Cluster Explorer, we can replace lines 12 to 38 with:
```TypeScript
    const kubectl = await k8s.extension.kubectl.v1;
    const cloudExplorer = await k8s.extension.cloudExplorer.v1;
    const clusterExplorer = await k8s.extension.clusterExplorer.v1;

    if (!kubectl.available) {
        vscode.window.showWarningMessage(`Kubectl is unavailable.`);
        return;
    }

    if (!cloudExplorer.available) {
        vscode.window.showWarningMessage(`Cloud explorer is unavailable.`);
        return;
    }

    if (!clusterExplorer.available) {
        vscode.window.showWarningMessage(`Cluster explorer is unavailable.`);
        return;
    }

    const clusterInfo = await getKubernetesClusterInfo(target, cloudExplorer, clusterExplorer);
    if (failed(clusterInfo)) {
        vscode.window.showErrorMessage(clusterInfo.error);
        return;
    }
```

(see, e.g. `aksKubectlCommands.ts`)